config/routes.rb

```diff
@@ -24,5 +24,5 @@
   get '/sign_up', to: "welcome#new"
   get '/deactivate_account', to: "welcome#deactivate_account"
   post '/deactivate', to: "welcome#deactivate"
-
+  post '/filter', to: "developers#filter"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.